packages/dev/repl/src/parcel/BrowserPackageManager.js

```diff
@@ -4,7 +4,7 @@ import type {FileSystem} from '@parcel/fs';
 import type {
   PackageManager,
   Invalidations,
-  ResolveResult,
+  PackageManagerResolveResult,
```

Comment:
This is also technically a breaking change yeah?

Reply:
Yeah... It is a breaking change. We can rename the re-export.

Reply:
I have fixed the breaking changes.